### PR TITLE
NeedActions singleton.

### PR DESCRIPTION
**Pull request by Mizipzor**
A cleaner implementation of the singleton pattern, not requiring someone to explicitly call the constructor.

### CODE REVIEW

File: Assets/Scripts/Models/World.cs [hunk 1]
```diff
@@ -167,7 +167,7 @@ void SetupWorld(int width, int height)
                 tiles[x, y].Room = GetOutsideRoom(); // Rooms 0 is always going to be outside, and that is our default room
             }
         }
-        new NeedActions ();
```
Comment (Mizipzor) [inline Assets/Scripts/Models/World.cs]: ...because this just feels wrong.

**Comment by NogginBops** [comment]
This should probably make use of the LuaUtilities class

**Comment by Mizipzor** [comment]
@NogginBops I think theres a separate pull request for that.

**Comment by NogginBops** [comment]
It might be my PR (If it is #837 you're thinking of)

**Pull request merged**